content/300-guides/050-database/830-prismas-type-system.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: How to use Prisma's type system
+metaDescription: How to use Prisma's type system
+tocDepth: 2
+---
+
+<TopBlock>
+
+This guide explains Prisma's type system and walks you through using it to introspect your database and apply schema changes.
+
+</TopBlock>
+
+## How does Prisma's type system work?
+
+Prisma uses types to define the kind of data that a field can hold. To make it easy to get started, Prisma provides a small number of core [scalar types](/reference/api-reference/prisma-schema-reference#model-field-scalar-types). For example, in the blog post model below, the `title` field uses the `String` scalar type, while the `createdAt` field uses the `DateTime` scalar type:
```

Comment:
```suggestion
Prisma uses _types_ to define the kind of data that a field can hold. To make it easy to get started, Prisma provides a small number of core [scalar types](/reference/api-reference/prisma-schema-reference#model-field-scalar-types) that should cover most default use cases. For example, in the blog post model below, the `title` field uses the `String` scalar type, while the `createdAt` field uses the `DateTime` scalar type:
```
- highlight first usage of `types`
- add some color to what we think the scalar types should cover